oshi-core/src/main/java/oshi/driver/windows/DeviceTree.java

```diff
@@ -91,7 +91,7 @@ private DeviceTree() {
         Map<Integer, String> mfgMap = new HashMap<>();
         // Get device IDs for the top level devices
         HANDLE hDevInfo = SA.SetupDiGetClassDevs(guidDevInterface, null, null, DIGCF_DEVICEINTERFACE | DIGCF_PRESENT);
-        if (hDevInfo != INVALID_HANDLE_VALUE) {
+        if (!hDevInfo.equals(INVALID_HANDLE_VALUE)) {
```

Comment:
better to have 
```suggestion
        if (!INVALID_HANDLE_VALUE.equals(hDevInfo)) {
```

see also below

Reply:
Switched it.